worker-languagedetection/README.md

```diff
@@ -24,8 +30,20 @@ Worker configuration is supported through the following environment variables:
     Default: `worker-out`  
     Sets the output queue where results are returned
 
+### Per Tenant Settings
+
+When `fieldSpecs` is provided to the worker via `customData` the worker's behaviour will change and the worker will begin [Multi-Field Processing](#multi-field-processing).
+
+ - `fieldSpecs`  
+    This field specifies to the worker on what fields to perform language detection. It is a string of comma-seperated values, each value will either be a complete field name or a partial field name supplied with a wildcard character "*".
```

Comment:
"This parameter specifies to the worker the fields on which to perform language detection." [try not to use "field" to refer to these settings as it could be confused with document fields]
"comma-separated..."
"each of which is either a complete field name ..."

Add that the wildcard character is supported anywhere in each value within the fieldSpecs parameter.